src/ai-bundle/config/services.php

```diff
@@ -111,6 +115,24 @@
         ->set('ai.platform.model_catalog.vertexai.gemini', VertexAiModelCatalog::class)
         ->set('ai.platform.model_catalog.voyage', VoyageModelCatalog::class)
 
+        // message templates
+        ->set('ai.platform.template_renderer.string', StringTemplateRenderer::class)
+            ->tag('ai.platform.template_renderer')
+        ->set('ai.platform.template_renderer.expression', ExpressionTemplateRenderer::class)
```

Comment:
lets check if expression language is available and then register this service and remove the nullOnINvalid()